metro.config.js

```diff
@@ -11,6 +11,14 @@ const defaultSourceExts = require('metro-config/src/defaults/defaults').sourceEx
 const defaultAssetExts = require('metro-config/src/defaults/defaults').assetExts
 
 module.exports = {
+  transformer: {
```

Comment:
this might look irrelevant, but i could not run the app without this change.

i upgraded react-native-svg so that css-select could be upgraded, which contains the security update for nth-check (the actual source of the vulnerability). it seems like css-select uses css-what under the hood, and one of these packages changed the way they are importing things. to be honest i'm a bit fuzzy on the details of why this react-native-svg upgrade caused issues, the error i was seeing was `TypeError: undefined is not an object (evaluating "css_what_1.SelectorType.Descendant")` which just yielded very little advice on google :/

Reply:
Was able create a release build with iOS 15.2 and Android 11.